docs/homepage.md

```diff
@@ -23,6 +23,12 @@ We do not store the secrets generated by this command and just rely on the secre
 
 ### Questions
 
+#### Why is this being added directly to the Signon landing page and via Publisher?
+
+The functionality itself will sit within Mainstream Publisher. However, we’ve made the decision to take an experimental approach and show the Homepage link within the main signon page.
+We’ve decided to trial this as we believe that users should simply be able to select what action they want to perform, rather than having to understand what app they need to use to accomplish a task.
```

Comment:
If we want these to be separate paragraphs, I think we need to add a blank line in the source code, otherwise they will all be rendered as a single paragraph.

Reply:
keeping it as one paragraph